### PR TITLE
Tn/automatic json compare

### PR DESCRIPTION
**Pull request by thiagonovaesb**

### CODE REVIEW

File: src/compare_json.jl [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    else #just a value
+        logs, ids = _deal_with_value(ids, d1, d2, adress, logs)
+    end
+    return logs, ids
+end
+
+function _deal_with_id_uniq(ids, d1, d2, key, adress, logs)
+    index1 = haskey(ids[1], d1)
+    index2 = haskey(ids[2], d2)
+    if (index1 == false) + (index2 == false) == 2 #check if they dont have the key
```
Comment (guilhermebodin) [inline src/compare_json.jl:18]: `if !index1 && !index2`

File: src/compare_json.jl [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        ids[1][d1] = adress
+        ids[2][d2] = adress
+    elseif (index1 == false) + (index2 == false) == 1 #check if one of then have the key
```
Comment (guilhermebodin) [inline src/compare_json.jl:21]: `if index1 || index2` ?

File: src/compare_json.jl (hunk 1 above)
Comment (thiagonovaesb) [reply src/compare_json.jl]: 158facae4eeda8f6a25ce255787b619646f7ba02

File: src/compare_json.jl (hunk 2 above)
Comment (thiagonovaesb) [reply src/compare_json.jl]: 158facae4eeda8f6a25ce255787b619646f7ba02

**Pull request closed without merging**